frontend/components/Header/Nav/SubNav.tsx

```diff
@@ -111,8 +118,10 @@ const parentStyle = css`
 `;
 
 interface Props {
-    parent?: LinkType;
-    links: LinkType[];
+    subnav?: {
+        parent?: LinkType | undefined;
```

Comment:
Do we need to specify this as a union type if it's an optional field? Asking for a friend 👀

Reply:
It's a good point actually. I just copied the original type definition, but I'm not sure it adds anything here.